packages/chai-openapi-response-validator/lib/assertions/satisfyApiSpec.js

```diff
@@ -44,7 +44,7 @@ function getExpectedResToSatisfyApiSpecMsg(
       'expected res to satisfy API spec' +
       `\n\nexpected res to satisfy a '${status}' response defined for endpoint '${endpoint}' in your API spec` +
       `\nres had request path '${requestPath}', but your API spec has no matching path` +
-      `\n\nPaths found in API spec: ${openApiSpec.paths().join(', ')}`;
+      `\n\nPaths found in API spec: ${openApiSpec.paths().map(path => (openApiSpec.spec.basePath || "")+path).join(', ')}`;
```

Comment:
could this OpenAPI 2- specific functionality be moved to `OpenAPI2Spec.js` by override its `paths()` method?

But first let's confirm what behaviour we want (see comment below)